api/rest/v1/getting_started.md

```diff
@@ -158,9 +172,114 @@ If your request are missing a required argument, you will receive an error messa
  ]
 }
 ```
+This is most commonly seen when a query parameter is misspelled or incorrect. Check the spelling of query parameters.
 
-If your request includes a bad argument, you'll receive an empty response like the following:
+## Key Terms
```

Comment:
We should add these to the glossary, and refer to that from here since these go beyond the V1 API.